src/main/build/debug.h

```diff
@@ -25,6 +25,7 @@ extern int16_t debug[DEBUG16_VALUE_COUNT];
 extern uint8_t debugMode;
 
 #define DEBUG_SET(mode, index, value) {if (debugMode == (mode)) {debug[(index)] = (value);}}
+#define DEBUG_INCR(mode, index) {if (debugMode == (mode)) {++debug[(index)];}}
```

Comment:
It's better to use do/while, so that the macro behaves like statement.
```
#define DEBUG_INCR(mode, index) do { if (debugMode == (mode)) {++debug[(index)];}} while(0)
```

Reply:
Macro removed